### PR TITLE
Update to upstream version 6.2.1

### PR DESCRIPTION
**Pull request by webwurst**

### CODE REVIEW

**Comment by webwurst** [comment]
Should we change the `[[ .Version ]]` in [Chart.yaml](https://github.com/giantswarm/grafana-app/blob/1aef5721a9f15fdc7357710a60e808a0ae7f9d96/helm/grafana-app/Chart.yaml#L28)?

**Comment by webwurst** [comment]
The Grafana pod went into the state `Blocked` after running with just the defaults in `values.yaml`. I overcome this I had to set:
```
rbac:
  pspUseAppArmor: false
```

**Comment by webwurst** [comment]
@piontec I get two warnings when running `dabs.sh` locally:
```
test_grafana.py:9
  /abs/workdir/helm/grafana/tests/abs/test_grafana.py:9: PytestUnknownMarkWarning: Unknown pytest.mark.flaky - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.flaky(reruns=10, reruns_delay=10)

test_grafana.py:10
  /abs/workdir/helm/grafana/tests/abs/test_grafana.py:10: PytestUnknownMarkWarning: Unknown pytest.mark.functional - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.functional
```
Are those fine to be ignored for now?

**Comment by webwurst** [comment]
When I run `pip install pytest-rerunfailures` inside the Docker container before `python -m app_build_suite [..]` the warnings are down to: `Unknown pytest.mark.functional`

When I uncomment `@pytest.mark.functional` for `test_grafana_login_page_available` the test seems to be not executed.

File: helm/grafana-app/values.yaml [hunk 1]
```diff
@@ -620,19 +631,21 @@ imageRenderer:
   enabled: false
   replicas: 1
   image:
-    registry: quay.io
     # image-renderer Image repository
-    repository: giantswarm/grafana-image-renderer
+    repository: grafana/grafana-image-renderer
     # image-renderer Image tag
-    tag: 2.0.0
+    tag: latest
```
Comment (ubergesundheit) [inline helm/grafana-app/values.yaml:637]: is this an upstream change?

Comment (webwurst) [reply helm/grafana-app/values.yaml:637]: Yes, right, should use our registry, will change that back.

**Comment by webwurst** [comment]
Running with `DABS_TAG=0.1.7` gives:
```
Creating a virtualenv for this project...
Pipfile: /abs/workdir/helm/grafana/tests/abs/Pipfile
Using /abs/.venv/bin/python3.8 (3.8.6) to create virtualenv...
⠋ Creating virtual environment...created virtual environment CPython3.8.6.final.0-64 in 1111ms
  creator CPython3Posix(dest=/abs/.local/share/virtualenvs/abs-ZZaIuwlU, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/abs/.local/share/virtualenv)
    added seed packages: pip==21.0.1, setuptools==52.0.0, wheel==0.36.2
  activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator

✔ Successfully created virtual environment! 
Virtualenv location: /abs/.local/share/virtualenvs/abs-ZZaIuwlU
Installing dependencies from Pipfile.lock (3fb8dd)...
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 17/17 — 00:00:17
2021-02-03 17:27:57,895 app_build_suite.utils.processes INFO: Command executed, exit code: 0.
2021-02-03 17:27:57,895 app_build_suite.build_steps.pytest INFO: Running pytest tool in './helm/grafana/tests/abs' directory.
2021-02-03 17:27:57,895 app_build_suite.utils.processes INFO: Running command:
2021-02-03 17:27:57,896 app_build_suite.utils.processes INFO: pipenv run pytest -m functional --cluster-type kind --kube-config /abs/workdir/kube.config --chart-path grafana-0.1.0-alpha1-b5f398946499c9494e94ff83d1623882ff0de1ec.tgz --chart-version 0.1.0-alpha1-b5f398946499c9494e94ff83d1623882ff0de1ec --chart-extra-info external_cluster_version=1.19.1 --log-cli-level info --junitxml=test_results_functional.xml
==================================================================================================================== test session starts ====================================================================================================================
platform linux -- Python 3.8.6, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /abs/workdir
plugins: helm-charts-0.3.1, rerunfailures-9.1.1
collected 1 item                                                                                                                                                                                                                                            

test_grafana.py::test_grafana_login_page_available 
---------------------------------------------------------------------------------------------------------------------- live log setup -----------------------------------------------------------------------------------------------------------------------
INFO     pytest_helm_charts.fixtures:fixtures.py:85 Cluster configured

--------------------------------------------------------------------------------------------------------------------- live log teardown ---------------------------------------------------------------------------------------------------------------------
INFO     pytest_helm_charts.fixtures:fixtures.py:91 Cluster released
PASSED                                                                                                                                                                                                                                                [100%]

===================================================================================================================== warnings summary ======================================================================================================================
test_grafana.py:10
  /abs/workdir/helm/grafana/tests/abs/test_grafana.py:10: PytestUnknownMarkWarning: Unknown pytest.mark.functional - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.functional

```

File: .abs/main.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-chart-dir: helm/grafana-app
+chart-dir: helm/grafana
```
Comment (ubergesundheit) [inline .abs/main.yaml:1]: If you don't rename the directory in the helm directory, this shouldn't be renamed either

**Review by piontec — APPROVED** [review]
Looks good, let's try it on circle

**Comment by ubergesundheit** [comment]
heads up IMO the change from `grafana-app` to just `grafana` will break the upgrade path

**Comment by webwurst** [comment]
Good point, will test that.

**Pull request merged**